### DISCUSSION TITLE
Segmented Downloading fails windows server 2016

### DISCUSSION THREAD

**Discussion by mstickler1** [iterate-ch/cyberduck#14489] [Q&A]
I have IIS FTP server running on Windows Server 2016. Cyberduck can connect and starts downloading fine but fails about 6mb into any file. If the client goes into their preferences and disables segmented transfers then downloads work fine.  Does anyone know if the segmented downloading can be disabled on a per connection bases (in the bookmark for their connections? ) and if so, how?

Or does anyone know if there's a setting on IIS/2016 that will allow connections to work?

**Comment by AliveDevil** [comment]
You can check the [documentation](https://learn.microsoft.com/en-us/iis/configuration/system.applicationhost/sites/sitedefaults/ftpserver/connections#attributes), that you didn't artificially limit the number of concurrent connections.

**Reply by AliveDevil** [reply]
Just checked with a Server 2016 image, installed IIS Role, added FTP Server, downloaded a 9 GiB file off of that server through Cyberduck, didn't encounter any issue (kept everything default).
Segments downloaded fine, and were correctly recombined.
